chain-signatures/node/src/protocol/message.rs

```diff
@@ -309,7 +309,7 @@ impl MessageHandler for RunningState {
             }
 
             let protocol = match presignature_manager
-                .get_or_generate(
+                .get_or_start_protocol(
```

Comment:
hmm, generate is basically start protocol though. I would rather we keep it as what we had before, because we also have the `generate` function

Reply:
Renamed to `get_or_start_generation`.
When you call `presignature_manager.get_or_generate` you are expecting a presignature, not a protocol.

Reply:
hmm, that's not right either though. `get_or_generate` never returned us the presignature, it always returned us the protocol